cranelift-filetests/src/function_runner.rs

```diff
@@ -1,53 +1,280 @@
+#![allow(trivial_numeric_casts)]
```

Comment:
Could you please move this to the actual function doing this?

Reply:
I have removed it instead.